libpromises/locks.c

```diff
@@ -431,6 +431,11 @@ static char *BodyName(const Promise *pp)
                 continue;
             }
 
+            if (strcmp(cp->lval, "arglist") == 0)      /* Exception for arglist, by symmetry, for locking */
+            {
+                continue;
+            }
+
```

Comment:
I don't quite understand why this needs to be here, but it seems preexisting, so better to follow the previous intent, I guess.